web.beans/src/org/netbeans/modules/web/beans/impl/model/Bundle.properties

```diff
@@ -14,6 +14,7 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+
```

Comment:
This is not necessary.

Reply:
Just trying to be coherent - some file has other not.

Reply:
Please revert this as well.

Reply:
@gilbertoca could you please address @junichi11 comment? This is not really a deal breaker, but we should try to not introduce unnessary changes. Thank you!

Reply:
I've already done at the same day - See the [Revert change](https://github.com/apache/incubator-netbeans/pull/694/commits/25cc426810c520e64afb3e58d3beeb70dbc1fd61) commit

Reply:
The linked revert covers:
`web.beans/src/org/netbeans/modules/web/beans/resources/Bundle.properties`
this file is:
`web.beans/src/org/netbeans/modules/web/beans/impl/model/Bundle.properties`
maybe you overlooked to commit the second file?

Reply:
I'm sorry, I will revert it as well